sdk/Javascript/src/Platforms/Platform.FxOS.js

```diff
@@ -0,0 +1,277 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+/// <reference path="..\Generated\MobileServices.DevIntellisense.js" />
+/*global $__fileVersion__:false, $__version__:false */
+
+var _ = require('Extensions');
+var Validate = require('Validate');
+var Promises = require('Promises');
+var Resources = require('Resources');
+var inMemorySettingStore = {};
+
+try {
+    var key = '___z';
+    localStorage.setItem(key, key);
+    localStorage.removeItem(key);
+    inMemorySettingStore = localStorage;
+} catch (e) {
+    // localStorage is not available
+}
+
+var bestAvailableTransport = null;
+var knownTransports = [ // In order of preference
+    require('DirectAjaxTransport'),
+    require('IframeTransport')
+];
+var knownLoginUis = [ // In order of preference
+    require('WebAuthBroker'),
```

Comment:
MozBrowser should be the default correct?  And you don't even need to list the other two?